src/main/groovy/de/dkfz/roddy/execution/jobs/direct/synchronousexecution/DirectSynchronousExecutionJobManager.groovy

```diff
@@ -118,6 +118,11 @@ class DirectSynchronousExecutionJobManager extends BatchEuphoriaJobManager<Direc
     }
 
     @Override
+    String getJobNameVariable() {
```

Comment:
$ is supposed to return a job name? How is this supposed to work?

Reply:
For the DirectSynchronousExecutionJobManager there is no good name for a process. So return the process ID ($$ in bash).